apps/financial-aid/web-osk/src/routes/application/TaxReturnForm/taxReturnForm.tsx

```diff
@@ -13,6 +13,35 @@ import useFormNavigation from '@island.is/financial-aid-web/osk/src/utils/hooks/
 
 import { NavigationProps } from '@island.is/financial-aid/shared/lib'
 
+const getContent = (fetchFailed: boolean) => {
```

Comment:
why is this not in utils or formatters? 🤔

Reply:
And why have the text component along since there is no difference between them? why not just below? 
```
   <Text marginBottom={2}>
        {content.data}
</Text> 

   <Text marginBottom={[4, 4, 5]}>
        {content.reason}
</Text> 
```

Reply:
Very legit questions.

1. Because we are using strong tag in the return values and then we would have to wrap them in <></>, instead of having needless fragments I decided to use what we actually use in the layout.


2. I have been thinking about this and feel we have been little on the line of making things exported when they are just used in a single place, like this will be used nowhere else so I feel it is better to use just here and not export it.

Reply:
Feel free do disagree 😄